heroku/cli
# migrate(W-14170470): Upgrade pg:credentials:create

This PR moves `pg:credentials:create` to the CLI and migrates to TypeScript

## Test
1. Checkout this branch and run `yarn && yarn build`
2. run `./bin/run pg:credentials:create -a YOUR_APP_WITH_PG -n NAME`
3. Observe output is the same as the existing cli
4. Repeat steps 2 and 3 using an app without PG
5. Repeat steps 2 and 3 using an app with a hobby PG
6. Repeat steps 2 and 3 using an app with a non-hobby pg
